crates/biome_cli/tests/snapshots/main_cases_protected_files/not_process_file_from_cli.snap

```diff
@@ -11,10 +11,16 @@ expression: content
 # Termination Message
 
 ```block
-internalError/io ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
```

Comment:
No longer calling it an internal error is such a subtle but great improvement of UX. Love it!